pkg/api/downstream/types/types.go

```diff
@@ -18,6 +19,7 @@ type Downstream struct {
 
 type DownstreamVersion struct {
 	VersionLabel             string                             `json:"versionLabel"`
+	Semver                   *semver.Version                    `json:"semver,omitempty"`
```

Comment:
why not just parse the version label during comparsion? having both Semver and VersionLabel feels off